src/test/resources/jsonpatch/extended/move?.json

```diff
@@ -0,0 +1,41 @@
+{
+    "errors": [
+        {
+            "op": { "op": "move?", "from": "/a", "path": "/b/c" },
+            "node": { "a": "b" },
+            "message": "jsonPatch.noSuchParent"
+        }
+    ],
```

Comment:
Could be worth adding tests for the other failure scenarios from the AddOperation like notAnIndex, noSuchParent, parentNotContainer.  Otherwise, this is good and on par with the regular move operation.